autocomplete.js

```diff
@@ -43,16 +43,26 @@ function getDate(paramName){
   return async (query, pluginSettings, pluginActionParams) => {
     const params = mapAutoParams(pluginActionParams);
     query = query || params[paramName];
-    let queryTime = new Date().toISOString().split("T")[0];
+   // let queryTime = new Date().toISOString().split("T")[0];
+    let date = new Date();
+    queryTime= (((date.getMonth() > 8) ? (date.getMonth() + 1) : ('0' + (date.getMonth() + 1))) + '/' + ((date.getDate() > 9) ? date.getDate() : ('0' + date.getDate())) + '/' + date.getFullYear());
```

Comment:
```suggestion
    const stringFormattedMonth = `${date.getMonth() > 8 ? "" : "0"}${date.getMonth() + 1}`;
    const stringFormattedDay = `${date.getDate() > 9 ? "" : "0"}${date.getDate() + 1}`;
    const queryTime= `${stringFormattedMonth}/${stringFormattedDay}/${date.getFullYear()}`;
```

We appreciate high intelectual skills of people who are able to understand such complex structures in a blink of an eye, but let's save their (and the less intelligent ones) brain computing power for serious engineering problems, and let's simplify common solutions.